src/process/types.rs

```diff
@@ -0,0 +1,84 @@
+#![allow(unsafe_code)]
+
+use crate::backend::c;
+use crate::pid::Pid;
+use core::mem::transmute;
+
+/// File lock data structure used in [`fcntl_getlk`].
+///
+/// [`fcntl_getlk`]: crate::fs::fcntl_getlk
+#[derive(Clone, Copy, Debug, PartialEq, Eq)]
+pub struct Flock {
+    /// Starting offset for lock
+    pub start: isize,
+    /// Number of bytes to lock
+    pub length: isize,
```

Comment:
Rustix exposes file sizes and offsets as 64-bit whenever possible, and follows [`std::io::SeekFrom`](https://doc.rust-lang.org/stable/std/io/enum.SeekFrom.html) in using `u64` for absolute offsets. Could you use `u64` for the start and length?